examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -35,9 +33,10 @@ export const containerConfig = {
 
 async function start(): Promise<void> {
     // Get or create the document depending if we are running through the create new flow
+    const containerConfig: ITinyliciousContainerConfig = { id: containerId };
```

Comment:
calling out the TinyliciousContainerConfig type here is a bit overkill since you don't do the same for the schema

Reply:
This was just to show that the package can be used to import both the config interface and the TinyliciousClient itself.
But agreed that it does make it verbose. Removed!